src/engine/menu/filenamer.lua

```diff
@@ -13,8 +13,10 @@ function FileNamer:init(limit, callback, name_text, confirm_text, default_name,
 
     self.default_name = default_name or ""
 
-    self.name_text    = name_text    or ("[style:GONER][spacing:3.2]ENTER YOUR OWN NAME.")
-    self.confirm_text = confirm_text or ("[style:GONER][spacing:3.2]THIS IS YOUR NAME.")
+    self.name_text    = Kristal.States.Menu.selected_mod.nameText or name_text    or ("[style:GONER][spacing:3.2]ENTER YOUR OWN NAME.")
+    self.confirm_text = Kristal.States.Menu.selected_mod.confirmText or confirm_text or ("[style:GONER][spacing:3.2]THIS IS YOUR NAME.")
+	
+    self.crash = Kristal.States.Menu.selected_mod.namesCrash or {"GASTER"}
```

Comment:
Make selected_mod optional, probably by saving it to another variable

```suggestion
    self.mod = Kristal.States.Menu.selected_mod or {}

    self.name_text    = self.mod.nameText or name_text    or ("[style:GONER][spacing:3.2]ENTER YOUR OWN NAME.")
    self.confirm_text = self.mod.confirmText or confirm_text or ("[style:GONER][spacing:3.2]THIS IS YOUR NAME.")
	
    self.crash = self.mod.namesCrash or {"GASTER"}
```